src/libYARP_OS/include/yarp/os/Time.h

```diff
@@ -69,13 +78,23 @@ class YARP_OS_API yarp::os::Time {
      *
      * \see yarp::os::NetworkClock
      *
+     * return true on success, false on failure. Possible causes of
+     * failure are invalid port name or address conflict.
+     *
+     * Throws assert in case of failure
```

Comment:
**Throws assert** is a bit strange phrase, as typically "exceptions" are thrown, but in this case I think the process exits in case `useNetworkClock` fails.